### PR TITLE
Metadata clean up for the articles

### PR DESCRIPTION
**Pull request by msewaweru**

### CODE REVIEW

**Comment by msewaweru** [comment]
> @msewaweru - What's your ETA on this PR? Thanks.

Hi @CelesteDG, looking to conclude this work within this week. Thanks.

File: microsoftgraph/docs-conceptual/app-only.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 title: "Use app-only authentication with the Microsoft Graph PowerShell SDK"
 description: "Learn how to use app-only authentication to enable non-interactive scenarios with the Microsoft Graph PowerShell SDK."
 
-ms.localizationpriority: medium
 author: jasonjoh
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/app-only.md:5]: @msewaweru Please add the manager field here. It's not so much direct manager as business owner, M1 PoC for the article in terms of reporting, escalation, etc. Can put me here and for the others with Jason as author. Thank you.

**Review by CelesteDG — CHANGES_REQUESTED** [review]
Just some minor tweaks. Thank you.

**Comment by msewaweru** [comment]
@CelesteDG, changes made and ready for merging. Thanks.

**Comment by msewaweru** [comment]
@mwongerapk, please merge this.

**Pull request merged**